<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvitationsDialog</class>
 <widget class="QDialog" name="InvitationsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Friend Invitations</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/icons/sun_icon.png</normaloff>:/resources/icons/sun_icon.png
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f5f6fa;
}

QTabWidget::pane {
    border: 2px solid #e1e8ed;
    border-radius: 8px;
    background-color: white;
}

QTabBar::tab {
    background-color: #f5f6fa;
    color: #2c3e50;
    padding: 8px 20px;
    border: 2px solid #e1e8ed;
    border-bottom: none;
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;
    font-family: 'Segoe UI';
    font-size: 13px;
    font-weight: 600;
}

QTabBar::tab:selected {
    background-color: white;
    border-bottom: none;
}

QTabBar::tab:!selected {
    margin-top: 2px;
}

QListWidget {
    background-color: white;
    border: 2px solid #e1e8ed;
    border-radius: 8px;
    padding: 5px;
    font-family: 'Segoe UI';
    font-size: 13px;
    font-weight: 600;
    outline: 0;
}

QListWidget::item {
    padding: 8px 5px;
    margin: 0px;
    border-radius: 0px;
    font-family: 'Segoe UI';
    font-size: 13px;
    font-weight: 600;
}

QListWidget::item:hover {
    background-color: #f0f2f5;
}

QListWidget::item:selected {
    background-color: #e3f2fd;
    color: #2c3e50;
}

QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 20px;
    font-family: 'Segoe UI';
    font-size: 14px;
    font-weight: bold;
    min-height: 30px;
    min-width: 80px;
}

QPushButton:hover {
    background-color: #2980b9;
}

QPushButton:pressed {
    background-color: #2475a7;
}

QScrollBar:vertical {
    border: none;
    background-color: #f0f2f5;
    width: 10px;
    border-radius: 5px;
}

QScrollBar::handle:vertical {
    background-color: #bdc3c7;
    border-radius: 5px;
}

QScrollBar::handle:vertical:hover {
    background-color: #95a5a6;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
    background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="receivedTab">
      <attribute name="title">
       <string>Received (0)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QListWidget" name="receivedList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="acceptButton">
           <property name="text">
            <string>Accept</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rejectButton">
           <property name="text">
            <string>Reject</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sentTab">
      <attribute name="title">
       <string>Sent (0)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QListWidget" name="sentList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
